src/main/kotlin/com/exactpro/th2/check1/utils/ProtoMessageUtils.kt

```diff
@@ -24,6 +23,9 @@ import com.exactpro.th2.common.grpc.Direction
 import com.exactpro.th2.common.grpc.MessageFilter
 import com.exactpro.th2.common.grpc.RootMessageFilter
 import com.exactpro.th2.sailfish.utils.ProtoToIMessageConverter
+import com.exactpro.th2.check1.entities.Checkpoint as InternalCheckpoint
+import com.exactpro.th2.check1.entities.CheckpointData as InternalCheckpointData
+import com.exactpro.th2.common.grpc.Checkpoint.CheckpointData as CheckpointData
```

Comment:
There is no need to use aliases for inner classes. They can be simple imported
```suggestion
import com.exactpro.th2.common.grpc.Checkpoint.CheckpointData
```